packages/hardhat-core/src/internal/hardhat-network/provider/provider.ts

```diff
@@ -165,7 +164,7 @@ export class EdrProviderWrapper
   public static async create(
     config: HardhatNetworkProviderConfig,
     loggerConfig: LoggerConfig,
-    tracingConfig?: TracingConfig
+    tracingConfig?: TracingConfigWithBuffers
```

Comment:
I was initially worried about this move as it is changing the API, but EdrProviderWrapper is in internal and is not exposed to the user. The only place create is invoked, uses `makeTracingConfig` which has had its return type altered as well.

Put another way this signature change looks safe to me.